README.md

```diff
@@ -27,10 +27,10 @@ After you're finished please remove all the comments and instructions!
 
 <div align="center">
   <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
-  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <img src="./banner.png" alt="logo" width=""  height="auto" />
```

Comment:
- Awesome work adding your customized banner! It adds a nice touch to your `README` and shows your creativity. It also helps to build your personal brand, which is always a good thing. 💪